### DISCUSSION TITLE
How to implement multiline ToggleGroup?

### DISCUSSION THREAD

**Discussion by roigreenberg** [unovue/shadcn-vue#937] [Q&A]
Hi.
I want to achieve something like in the picture.
It should be a general component that can have different items every time.

I tried to use ToggleGroup, but when I set many items, I get it all in 1 long line.
How can I make it so it will take just the container width, and start a new line when needed?

![image](https://github.com/user-attachments/assets/70b17f72-2fe2-430c-a0df-67c7e2d1cd3c)

**Comment by sadeghbarati** [comment]
Hi

`ToggleGroup` uses `flex` class, if you want to have multiline ToggleGroup without `flex-wrap` you can use `grid grid-cols-3` classes on the `ToggleGroup` component

https://stackblitz.com/edit/uq6gah?file=src%2FApp.vue

**Reply by roigreenberg** [reply]
Apparently, `flex-wrap` seems to do the job :)
